src/schemas/wait.json

```diff
@@ -1,57 +1,113 @@
 {
   "$id": "http://asl-validator.cloud/wait.json#",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "type": "object",
-  "properties": {
+  "definitions": {
+    "Seconds": {
+      "type": "number",
+      "minimum": 0
+    },
+    "Timestamp": {
+      "type": "string",
+      "format": "date-time"
```

Comment:
From the [docs](https://www.npmjs.com/package/ajv-formats)
> date-time: date-time (time-zone is mandatory).

Is timezone mandatory as well in ASL?

Reply:
Yes. See here: https://states-language.net/#timestamps

>The Choice and Wait States deal with JSON field values which represent timestamps. These are strings which MUST conform to the [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) profile of ISO 8601, with the further restrictions that an uppercase "T" character MUST be used to separate date and time, and an uppercase "Z" character MUST be present in the absence of a numeric time zone offset, for example "2016-03-14T01:59:00Z".